code/Model/EditableFormField/EditableCountryDropdownField.php

```diff
@@ -84,7 +84,7 @@ public function getFormField()
 
     public function getValueFromData($data)
     {
-        if (isset($data[$this->Name])) {
+        if (isset($data[$this->Name]) && !empty($data[$this->Name])) {
```

Comment:
```suggestion
        if (!empty($data[$this->Name])) {
```

Reply:
How have I been developing in PHP for over 10 years and only just found this out....🤦‍♂️